web/src/containers/ApdExport.js

```diff
@@ -0,0 +1,82 @@
+import { Button } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Waypoint from './ConnectedWaypoint';
+import { printApd } from '../actions/print';
+import { Section } from '../components/Section';
+import { FileDownload } from '../components/Icons';
+
+const Bold = ({ children }) => (
+  <span className="ds-u-font-weight--bold">{children}</span>
+);
+Bold.propTypes = { children: PropTypes.node.isRequired };
+
+const Italic = ({ children }) => (
+  <span className="ds-u-font-style--italic">{children}</span>
+);
+Italic.propTypes = { children: PropTypes.node.isRequired };
+
+const ExportAndSubmit = ({ printApd: print }) => (
+  <Waypoint id="export-and-submit">
+    <Section isNumbered id="export-and-submit" resource="exportAndSubmit">
+      <p>
+        To submit your APD for analyst review, export the document as a PDF,
+        then send it to CMS just like you’ve done before.
+      </p>
+      <ol>
+        <li>
+          Select <Bold>Export</Bold> to open up your browser’s print options.
```

Comment:
Y'know, we _could_ just use `strong` and `em` tags here and not bother with the helper functions (although if you'd rather use the span with the style, I really like that method of ensuring consistency!)

Reply:
Back in the old days, it'd have been `<b>` and `<i>` but it got drilled into my head not to use those.  Now `<strong>` and `<em>` are okay I guess?  But my brain still tells me never to use things like that.  😛

But if I switched to `<strong>` and `<em>`, we could get consistency by targeting those with CSS and setting the properties the same as the design system utility classes, right?